scipy/special/setup.py

```diff
@@ -159,6 +159,9 @@ def configuration(parent_package='',top_path=None):
 
     config.add_subpackage('_precompute')
 
+    # Type stubs
```

Comment:
Before I was running mypy against the source tree. Now that it's being run against the build tree we need to make sure these type stubs are getting installed.

Reply:
LGTM, and I verified that these also end up in the sdist after running `python setup.py sdist`.